scripts/update_meilisearch_version.sh

```diff
@@ -225,82 +225,88 @@ chmod +x meilisearch
 
 # Keep cache of previous data.ms in case of failure
 cp -r /var/lib/meilisearch/data.ms /tmp/
-echo "${INFO_LABEL}Keep a temporary copy of previous data.ms."
+echo -e "${INFO_LABEL}Keep a temporary copy of previous data.ms."
 
 # Remove data.ms
 rm -rf /var/lib/meilisearch/data.ms
-echo "${INFO_LABEL}Delete current Meilisearch's data.ms"
+echo -e "${INFO_LABEL}Delete current Meilisearch's data.ms"
 
 ## Move new Meilisearch binary to the systemctl directory containing the binary
-echo "${INFO_LABEL}Update Meilisearch version."
+echo -e "${INFO_LABEL}Update Meilisearch version."
 cp meilisearch /usr/bin/meilisearch
 
 # Run Meilisearch
 # TODO: `import-dump` may change name for v1, it should be added in the integration-guide issue
 # https://github.com/meilisearch/meilisearch/issues/3132
 ./meilisearch --db-path /var/lib/meilisearch/data.ms --env production --import-dump "/var/opt/meilisearch/dumps/$dump_id.dump" --master-key $MEILISEARCH_MASTER_KEY 2>logs &
-echo "${INFO_LABEL}Run local $meilisearch_version binary importing the dump and creating the new data.ms."
+echo -e "${INFO_LABEL}Run local $meilisearch_version binary importing the dump and creating the new data.ms."
 
 sleep 2
 
 # Needed conditions due to bug in Meilisearch #1701
 if cat logs | grep "Error: No such file or directory (os error 2)" -q; then
     # If dump was empty no import is needed
-    echo "${SUCCESS_LABEL}Empty database! Importing of no data done."
+    echo -e "${SUCCESS_LABEL}Empty database! Importing of no data done."
 else
-    echo "${INFO_LABEL}Check if local $meilisearch_version started correctly."
+    echo -e "${INFO_LABEL}Check if local $meilisearch_version started correctly."
     # Check if local meilisearch started correctly `./meilisearch ..`
     if ps | grep "meilisearch" -q; then
-        echo "${SUCCESS_LABEL}Meilisearch started successfully and is importing the dump."
+        echo -e "${SUCCESS_LABEL}Meilisearch started successfully and is importing the dump."
     else
-        echo "${ERROR_LABEL}Meilisearch could not start: \n ${BRED}$(cat logs)${NC}." >&2
+        echo -e "${ERROR_LABEL}Meilisearch could not start: \n ${BRED}$(cat logs)${NC}." >&2
         # In case of failed start rollback to initial version
         previous_version_rollback
     fi
 
     ## Wait for pending dump indexation
     #FIXME: Avoid infinite loop see issue #45
     until curl -X GET 'http://localhost:7700/health' -s >/dev/null; do
-        echo "${PENDING_LABEL}Meilisearch is still indexing the dump."
+        echo -e "${PENDING_LABEL}Meilisearch is still indexing the dump."
         sleep 2
     done
-    echo "${SUCCESS_LABEL}Meilisearch is done indexing the dump."
+    echo -e "${SUCCESS_LABEL}Meilisearch is done indexing the dump."
 
     # Kill local Meilisearch process
-    echo "${INFO_LABEL}Kill local Meilisearch process."
+    echo -e "${INFO_LABEL}Kill local Meilisearch process."
     pkill meilisearch
 fi
 
+regex_version="v([1-9]+\.[0-9]+\.[0-9])"
```

Comment:
```suggestion
regex_version="v([1-9]+\.[0-9]+\.[0-9]+)"
```

Is `meilisearch_version` formated like this `v0.0.0` or like this `0.0.0` ? In which case you can remove the `()`

Reply:
Like this `v0.0.0` that why there is the `v`